pkg/ccl/backupccl/restore_job.go

```diff
@@ -1689,7 +1742,9 @@ func (r *restoreResumer) doResume(ctx context.Context, execCtx interface{}) erro
 			return err
 		}
 	}
-
+	if err := r.execCfg.ProtectedTimestampManager.Unprotect(ctx, r.job.ID()); err != nil {
+		return err
```

Comment:
What happens to the restore if this fails?

Reply:
good point. Instead we now log a warning, to follow in backup's footsteps. I believe there's some background reconciler that will remove the pts if this thing fails as well.